VastClient/VastClient/constants/VastErrorCodes.swift

```diff
@@ -16,15 +16,20 @@ public enum VastErrorCodes: Int {
     case videoPlayerExpectingDifferentLinearity = 201
     case videoPlayerExpectingDifferentDuration = 202
     case videoPlayerExpectingDifferentSize = 203
+    case adCategoryNotProvided = 204
     case generalWrapperError = 300
     case timeoutOfVastURI = 301
     case wrapperLimitReached = 302
     case noAdsVastResponse = 303
+    case inlineResponseFailedToDisplayInTime = 304
     case generalLinearError = 400
     case fileNotFound = 401
     case timeoutOfMediaFile = 402
     case mediaFileNotSupported = 403
     case problemDisplayingMediaFile = 405
+    case conditionalAdRejected = 408
+    case interactiveUnitInNodeNotExecuted = 409
+    case verificationUniInNodeNotExecuted = 410
```

Comment:
Typo in `verificationUniInNodeNotExecuted`? -> `verificationUnitInNodeNotExecuted`